app/src/test/java/org/simple/clinic/bp/assignbppassport/BpPassportEffectHandlerTest.kt

```diff
@@ -81,4 +81,27 @@ class BpPassportEffectHandlerTest {
     verifyNoMoreInteractions(uiActions)
   }
 
+  @Test
+  fun `when send selected button name effect is received, then send add to existing patient`() {
```

Comment:
Looks like you forgot to rename this test to reflect the effect name change. Also one test should be enough for this scenario since the effect takes a sealed class. So we don't have test each option like this.